assets/src/dashboard/theme.js

```diff
@@ -107,6 +107,48 @@ const theme = {
       letterSpacing: '0.01em',
     },
   },
+  grid: {
+    desktop: {
+      width: '1280px',
+      columns: 5,
+      gap: '25px',
+      itemWidth: '221px',
+      itemHeight: '453px',
+    },
+    tablet: {
+      width: '1120px',
+      columns: 5,
+      gap: '22px',
+      itemWidth: '189px',
+      itemHeight: '395px',
+    },
+    mobile: {
+      width: '800px',
+      columns: 4,
+      gap: '20px',
+      itemWidth: '162px',
+      itemHeight: '347px',
+    },
+    min: {
+      width: '684px',
+      columns: 3,
+      gap: '24px',
+      itemWidth: '185px',
+      itemHeight: '395px',
+    },
+  },
+  breakpoint: {
+    desktop: 'screen and (max-width: 1440px)',
+    tablet: 'screen and (max-width: 1280px)',
+    mobile: 'screen and (max-width: 960px)',
```

Comment:
The navigation elements break into its mobile design at 800px, should we change `960` to `800`?

Reply:
100% - I started with this and then saw the breakpoints within the whole page on figma and noticed that the page width vs. the grid width were different (where the grid's 800px when the breakpoint for the page is 960 given the sidenav) but now that i'm going back for that documentation I can't find it. I'll make the update because if the nav elements break at 800 then that's the right thing. Thanks for flagging this.